test_scripts/API/PerformAudioPassThru/ideas_PAPT_new_param_audioPassThruIcon.txt

```diff
@@ -0,0 +1,56 @@
+  +./test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua
+  Check_that_INVALID_DATA_returned_when_audioPassThruIcon_has_invalid_value_parameter_valid_mandatory_params_sent_DYNAMIC_image_type 
+  +./test_scripts/API/PerformAudioPassThru/017_ATF_N_PerformAudioPassThru_INVALID_DATA_All_Params_audioPassThruIcon_invalid_ImageType.lua
+  Check_that_INVALID_DATA_returned_when_audioPassThruIcon_has_invalid_ImageType_parameter_all_other_params_valid  
+  +./test_scripts/API/PerformAudioPassThru/018_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_invalid_ImageType.lua
+  Check_that_INVALID_DATA_returned_when_audioPassThruIcon_has_invalid_ImageType_parameter_mandatory_params_valid
+  +./test_scripts/API/PerformAudioPassThru/019_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_AllParams_STATIC.lua 
+  Check_that_WARNINGS_returned_when_audioPassThruIcon_missing_in_folder_all_params_STATIC
+  +./test_scripts/API/PerformAudioPassThru/020_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_MandatoryParams_STATIC.lua 
+  Check_that_WARNINGS_returned_when_audioPassThruIcon_missing_in_folder_mandatory_params_STATIC
+  +./test_scripts/API/PerformAudioPassThru/021_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_AllParams_DYNAMIC.lua
+  Check_that_WARNINGS_returned_when_audioPassThruIcon_missing_in_folder_all_params_DYNAMIC
+  +./test_scripts/API/PerformAudioPassThru/022_ATF_P_PerformAudioPassThru_WARNING_Missing_audioPassThruIcon_MandatoryParams_DYNAMIC.lua 
+  Check_that_WARNINGS_returned_when_audioPassThruIcon_missing_in_folder_mandatory_params_DYNAMIC  
+  +./test_scripts/API/PerformAudioPassThru/023_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_false.lua 
+  Check_that_SDL_returns_resultcode_from_UI_when_TTS_returns_SUCCESS_UI_loop_resultcodes_success_false 
+  +./test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua 
+  Check_that_SDL_returns_resultcode_from_UI_when_TTS_returns_SUCCESS_UI_loop_resultcodes_success_true
+  +./test_scripts/API/PerformAudioPassThru/025_ATF_P_PerformAudioPassThru_UI_SUCCESS_TTS_success_false_true.lua 
+  Check_that_SDL_returns_WARNINGS_when_UI_returns_SUCCESS_TTS_loop_resultcodes_success_false_true
+  +./test_scripts/API/PerformAudioPassThru/026_ATF_P_PerformAudioPassThru_UI_success_false_TTS_success_false.lua 
+  Check_that_SDL_returns_resultcode_from_UI_when_loop_UI_returns_one_errorcode_TTS_returns_another_error_code
+  +./test_scripts/API/PerformAudioPassThru/027_ATF_P_PerformAudioPassThru_UI_timeout_TTS_success.lua 
+  Check_that_SDL_returns_GENERIC_ERROR_when_HMI_does_not_reply_to_UI
+  +./test_scripts/API/PerformAudioPassThru/028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua 
+  Check_that_SDL_returns_GENERIC_ERROR_when_HMI_does_not_reply_to_TTS

```

Comment:
@maria-mih, I can't see tests for requirements: 
"[GENERIC_ERROR]: SDL behavior in case HMI sends invalid response AND SDL must transfer this response to mobile app"
"[HMI RPC validation]: SDL must send GENERIC_ERROR to mobile app in case SDL cuts off fake params from RPC and this RPC becomes invalid"
"[HMI RPC validation]: SDL behavior: HMI sends request (response, notification) with fake parameters that SDL should transfer to mobile "